docs/application/dotnet/guides/app-management/widget-control.md

```diff
@@ -0,0 +1,149 @@
+
+# Widget Control
+
+It also provides functions for the following:
+
+*   Sending update requests to the widget applications
+*   Retrieving details of running instance for the same package widget application
```

Comment:
Please change "application" to "applications"

Reply:
Done